test/Governance.test.ts

```diff
@@ -279,19 +290,16 @@ describe('Governance', function () {
 
     it('Verify initial governance state', async function () {
         expect(
-            await this.governance.votersRoles(
+            await this.governance.hasRole(
                 DEVELOPER_ROLE,
                 this.delegatee1.address
             )
         ).to.equal(true)
         expect(
-            await this.governance.votersRoles(
-                LEGAL_ROLE,
-                this.delegatee2.address
-            )
+            await this.governance.hasRole(LEGAL_ROLE, this.delegatee2.address)
         ).to.equal(true)
         expect(
-            await this.governance.votersRoles(
+            await this.governance.hasRole(
```

Comment:
question: does that method name change in this PR?

Not spotting a function name change in this PR. A logic change?

Reply:
It was changed before during some refactoring before (made getters instead of storage access), which was merged, but it was not changed in broad tests checks PR as it was separate PR and its one of the reasons why pipeline got broken.

Reply:
Cool 👌